pom.xml

```diff
@@ -138,7 +138,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>8.0.22</version>
+                <version>8.0.23</version>
```

Comment:
Currently RDS is version capped to .21. MAT updated this dependency to .23 and ran into liquibase issues. Since ms doesn't use liquibase, might be fine.